src/style.css

```diff
@@ -1,3 +1,95 @@
+/* reset browser default margins */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: var(--main-collor) #000;
+  background-color: #fff;
+}
+/* #region header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__link {
+  color: var(--main-collor);
+  text-decoration: none;
+  display: block;
+  padding: 10px 0;
+}
+
+.header__logo {
+  width: 40px;
+  margin: auto;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  height: fit-content;
+  justify-content: space-between;
+}
+
+li {
+  margin: 0;
+  height: fit-content;
+}
+
+.nav__link {
+  display: block;
+  color: var(--main-collor);
+  text-decoration: none;
+  padding: 22.5px 0;
```

Comment:
instead of centering the text of this link with padding, better do it with flexbox